rubix-bookkeeper/src/test/java/com/qubole/rubix/bookkeeper/validation/TestFileValidatorVisitor.java

```diff
@@ -53,7 +77,10 @@ public void tearDown() throws IOException
     CacheConfig.setCacheDataDirPrefix(conf, TEST_CACHE_DIR_PREFIX);
     TestUtil.removeCacheParentDirectories(conf, TEST_MAX_DISKS);
 
+    BaseServerTest.stopBookKeeperServer();
+
     conf.clear();
+    metric = null;
```

Comment:
`metric` is reassigned at the start of every test, so this shouldn't be necessary. However, this does make it clearer that the metrics registry is reset every time.